pkg/test/istio.io/README.md

```diff
@@ -1,13 +1,42 @@
-# Istio.io Example Tests
+# Istio.io preliminary test cases
+
+Welcome to the development of the istio.io preliminary test cases!
+
+This work is experimental. This work is also under construction and may not work
+for you. The Istio developers have done some work to document known problems or
+workarounds. We expect this work will take several releases to complete.
 
 ## Overview
-This folder contains tests for the different examples on Istio.io.
+This folder contains tests for the different examples on [istio.io](https://istio.io).
+The goal of this work is to ensure [istio.io](https://istio.io) is tested according
+to the code in the [Istio repository](https://github.com/istio).
 
 ## Writing Tests
 
+This section needs fleshing out.
```

Comment:
For the record, I think it would be a good practice to create an `example.New` for every top section in a doc page. `Example`s are very lightweight, so it's good to create more of those to structure a test case.

Also, all runtime error logs for an example are showing up as caused by the one line that calls `Example.Run` for this example. Therefore, splitting into multiple examples better helps localizing the cause of runtime errors.

Reply:
Every package creates a new instance of Istio and is pretty heavy. So if you can have example share one install it is best to place them in the same package

Reply:
I was talking specifically about the `examples.New` and `Example` type that we use here. I was not discussing creating one test suite for every section.